install.sh

```diff
@@ -80,7 +80,36 @@ file_not_grpowned() {
   [[ " $(id -G "${USER}") " != *" $(get_group "$1") "* ]]
 }
 
-BUGSNAG_CLI_GIT_REMOTE="https://api.github.com/repos/bugsnag/bugsnag-cli/releases/latest"
+display_help() {
+  echo ""
+  cat <<EOS
+  Usage: ./$0
```

Comment:
This outputs the following when you `./` the script:

```
  Usage: ././install.sh
```

Any reason not remove the "./" in the script or just to hard-code `./install.sh`?